worker/proxyupdater/proxyupdater_test.go

```diff
@@ -171,6 +172,25 @@ func (s *ProxyUpdaterSuite) waitForFile(c *gc.C, filename, expected string) {
 	}
 }
 
+func (s *ProxyUpdaterSuite) assertNoFile(c *gc.C, filename string) {
+	//TODO(bogdanteleaga): Find a way to test this on windows
+	if runtime.GOOS == "windows" {
+		c.Skip("Proxy settings are written to the registry on windows")
```

Comment:
Shouldn't this skip be in the test itself?

Reply:
Probably, but I was copying a pattern in the other functions.